python/cuml/preprocessing/text/stem/porter_stemmer_utils/suffix_utils.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
```

Comment:
Since this code is just going in, I think we can just keep 2020 in all of these files.

Reply:
Resolved by https://github.com/rapidsai/cuml/pull/2476/commits/d95944958c7be3d80810d13e98281d1e70c7f238 .